src/DatabaseProvider/index.js

```diff
@@ -1,9 +1,7 @@
 // @flow
 import React from 'react'
 import Database from '../Database'
-
-export const DatabaseContext = (React.createContext(): any)
-const { Provider, Consumer } = DatabaseContext
+import { Provider } from './DatabaseContext'
```

Comment:
can you export DatabaseContext from here? Otherwise, you're making an API-breaking change, in case someone is using this export